### PR TITLE
Follow up to Topology Manager PR

### PR DESCRIPTION
**Pull request by mburke5678**
Editorial followup to https://github.com/openshift/openshift-docs/pull/22663

### CODE REVIEW

**Comment by mburke5678** [comment]
@vpickard Can you take a look at my edits to make sure I didn't break anything?

File: modules/setting-up-topology-manager.adoc [hunk 1]
```diff
@@ -52,14 +55,17 @@ metadata:
...
-
-
+  featureSet: LatencySensitive <1>
+...
 ----
+<1> Add the `LatencySensitive` FeatureGate in a comma-separated list.
 
-. Configure the Topology Manager policy with KubeletConfig.
+. Edit the `cpumanager-enabled` custom resource (CR) to configure the Topology Manager policy.
+Create the file if it does not exist. Creating the file also activates CPU Manager.
```
Comment (mburke5678) [inline modules/setting-up-topology-manager.adoc:64]: @vpickard Am I correct here? The user edits the CPU Manager policy file, if it exists? If not, the user needs to add the parameters to create the CPU manager?

Comment (mburke5678) [reply modules/setting-up-topology-manager.adoc:64]: Note my question at https://github.com/openshift/openshift-docs/pull/22756/files#r436923741

Comment (Deepthidharwar) [reply modules/setting-up-topology-manager.adoc:64]: Yes, that's right.

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions, but this largely looks good.

**Pull request merged**